src/finder/finderUtil.ts

```diff
@@ -0,0 +1,28 @@
+export function extractSubIncludedText(content: string, tag: string): string {
+  const STARTSUB_REGEX = /^\s*!startsub\s+(.*)\s*$/;
+  const ENDSUB_REGEX = /^\s*!endsub\s*$/;
+
+  const contentLines = content.split('\n');
+  const subTextLines = [];
+  let subDepth = 0;
+  for (const line of contentLines) {
+    if (subDepth === 0) {
+      const match = STARTSUB_REGEX.exec(line);
+      if (match !== null && match[1] === tag) subDepth++;
```

Comment:
I think this should be `if (match && match[1] === tag)` (and similar conditinos should be simplified).